packages/chrome/src/elements/Chrome.spec.tsx

```diff
@@ -18,6 +18,20 @@ describe('Chrome', () => {
     expect(container.firstChild).toBe(ref.current);
   });
 
+  it('renders fluid html element', () => {
+    render(<Chrome isFluid />);
+
+    expect(document.querySelector('html')).toHaveAttribute('style', '');
```

Comment:
[nit] I think you could also use `...'html')).not.toHaveAttribute('style');`

Reply:
Hmm that didn't work.

Reply:
🤷 fine as-is.